samples/nrf9160/nrf_cloud_mqtt_multi_service/src/connection.c

```diff
@@ -702,12 +709,20 @@ void manage_connection(void)
 	 * (Once we request connection, the modem will automatically try to reconnect whenever
 	 *  connection is lost).
 	 */
-
-	LOG_INF("Connecting to LTE network. This may take several minutes...");
+	LOG_INF("Setting up LTE...");
 	if (setup_lte()) {
-		LOG_ERR("LTE initialization failed. Continuing anyway. This may fail.");
+		LOG_ERR("LTE setup failed. Continuing anyway.");
```

Comment:
Same as below... maybe try again?  Continuing after this failure will probably just lead to more errors.

Reply:
that link doesn't work for me.

so does this current method achieve the goal of error recovery?

Reply:
Sorry, here is a working link:
https://github.com/nrfconnect/sdk-nrf/pull/7056#discussion_r849135826

The point is that I don't think error recovery is possible in this case

Reply:
so there is no point to continue with running the application then... except to produce more error messages.

Reply:
I am not opposed to just giving up

Reply:
that's the spirit! 🎉

Reply:
Actually, when does this function fail?  What if the device is locked in the RF chamber?  This mimics what happens when a device is out in the boonies and has no cell connection, and then eventually gets in range of one.  Will it recover in that case, or still be hung if you make this change?

Reply:
Yep, `setup_lte()` completes with no errors when inside RF chamber.

I guess it's not explicitly stated anywhere what the behavior of `lte_lc_init_and_connect_async` will be if the first connection attempt fails, although the rule of thumb I've heard is that "it's basically just like a cell phone". So, if that rule of thumb holds true, it should eventually connect once the RF chamber is opened.

I will leave the board in the RF chamber for a day or two, presuming I don't flash to it by accident